source/module_base/test/global_file_test.cpp

```diff
@@ -0,0 +1,32 @@
+#include "../global_file.h"
+#include "../global_variable.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include <fstream>
+#include <unistd.h>
+
+#ifdef __MPI
+#include "mpi.h"
+#endif
+
+/************************************************
+ *  unit test of functions in global_file.cpp
+ ***********************************************/
+
+/**
+ * - Tested Function
+ * - mkdiratom
+ *   - generate atom dir for each type of atom
+ */
+
+class GlobalFile : public testing::Test
+{
+
+};
+
+TEST_F(GlobalFile,mkdiratom)
+{
+        ModuleBase::Global_File::make_dir_atom("Si");
```

Comment:
in order to use GlobalV, you need to include the corresponding header file. Could you please try to set the output directory, for example to "./" ?